docs/pages/access-controls/guides/headless.mdx

```diff
@@ -140,14 +150,30 @@ $ tsh ls --headless --proxy=proxy.example.com --user=alice
 # Node Name Address        Labels                                                                             
 # --------- -------------- -----------
 # server01  127.0.0.1:3022 arch=x86_64
-#
-$ tsh ssh --headless --proxy=proxy.example.com --user=alice server01
+```
+
+### SSH Session Example:
+```code
+$ tsh ssh --headless --proxy=proxy.example.com --user=alice alice@server01
 # Complete headless authentication in your local web browser:
 # 
 # https://proxy.example.com:3080/web/headless/864cccd9-2425-46d9-a9f2-636387e66ebf
 # 
 # or execute this command in your local terminal:
 # 
 # tsh headless approve --user=alice --proxy=proxy.example.com 864cccd9-2425-46d9-a9f2-636387e66ebf
-# # User approves through link
-```
\ No newline at end of file
+# # User approves through link and a ssh terminal starts
+alice@server01 $
+```
+
+<Notice type="note">
```

Comment:
I would suggest using an Admonition—or ideally regular body text—instead of this Notice since the text is quite long here.